README.md

```diff
@@ -26,38 +26,144 @@ guarantee these changes will be backwards-compatible.
   - [User Stories](https://docs.google.com/document/d/1wT8e8wBHMrSRHY4UF_60GCgyWGqvYye4THvaDARPySs/edit)
   - [Features](https://docs.google.com/document/d/1JNP6DgSK-c6KqveIhQk-n_HAw3hsZkL-okoleM43NgA/edit)
   - [Design Space](https://docs.google.com/document/d/1TpQlQYovCoX9FkpgsoxzdvZplghudHAiQOame30A-v8/edit)
-- [Frequently Asked Questions](FAQ.md)
+- [Frequently Asked Questions](docs/FAQ.md)
 
-## Usage
+## Setup
 
 Get the tool via
 
 ```sh
 $ go get -u github.com/golang/dep/cmd/dep
 ```
 
-Typical usage on a new repo might be
+To start managing dependencies using dep, run the following from your project root directory:
 
 ```sh
 $ dep init
 $ dep ensure -update
```

Comment:
Now that the instructions above have changed from "here are typical commands" to "Do this to start using dep", we should remove `dep ensure -update`.

All you need to start using dep is init, and if they had specific versions identified by dep either from their GOPATH, or other dependency managers, we don't want to immediately clobber them. 😀